packages/webapp/src/components/Finances/DateRangeSelector/index.jsx

```diff
@@ -78,4 +80,8 @@ const FinanceDateRangeSelector = () => {
   );
 };
 
+FinanceDateRangeSelector.PropTypes = {
```

Comment:
There is a warning about the "P" in "PropTypes" being capitalized!

Reply:
Thank you!